{"uuid":"7b0dcc9b-06ea-4aa5-9820-84df5c9c25f7","historyId":"829f4964dda0ff3174a63b52853b158c","fullName":"Array: The user is able to submit code in tryEditor for Max Consecutive Ones","labels":[{"name":"tag","value":"arrays"},{"name":"host","value":"LAPTOP-S7F1L181"},{"name":"thread","value":"7872@LAPTOP-S7F1L181.main(1)"},{"name":"feature","value":"Array"},{"name":"story","value":"The user is able to submit code in tryEditor for Max Consecutive Ones"},{"name":"suite","value":"Array"},{"name":"testClass","value":"The user is able to submit code in tryEditor for Max Consecutive Ones"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/xunee/eclipse-workspace-2022/DS-ALGO-/src/test/resources/features/5arrays.feature"},{"name":"package","value":"src.test.resources.features.5arrays_feature.Array"}],"links":[],"name":"The user is able to submit code in tryEditor for Max Consecutive Ones","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Result do not match expected [submission success] but found [ Some Tests failed. Please review code]","trace":"java.lang.AssertionError: Result do not match expected [submission success] but found [ Some Tests failed. Please review code]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat com.stepDefinations.ArraysStepDef.the_user_should_be_presented_with_successful_submission_message(ArraysStepDef.java:160)\r\n\tat ✽.The user should be presented with successful submission message(file:///C:/Users/xunee/eclipse-workspace-2022/DS-ALGO-/src/test/resources/features/5arrays.feature:130)\r\n"},"stage":"finished","steps":[{"name":"Given  The user is on \"Question\" page of \"Max Consecutive Ones\" after logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673284084532,"stop":1673284084688},{"name":"When  The user enter valid python code in tryEditor from sheet \"pythonCode\" and 5 for the question","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673284084688,"stop":1673284086120},{"name":"And  The user clicks on Submit button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673284086120,"stop":1673284086167},{"name":"Then  The user should be presented with successful submission message","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Result do not match expected [submission success] but found [ Some Tests failed. Please review code]","trace":"java.lang.AssertionError: Result do not match expected [submission success] but found [ Some Tests failed. Please review code]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat com.stepDefinations.ArraysStepDef.the_user_should_be_presented_with_successful_submission_message(ArraysStepDef.java:160)\r\n\tat ✽.The user should be presented with successful submission message(file:///C:/Users/xunee/eclipse-workspace-2022/DS-ALGO-/src/test/resources/features/5arrays.feature:130)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673284086167,"stop":1673284086712}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"5"}],"start":1673284084531,"stop":1673284086884}